src/components/elements/SidebarGroup/SidebarGroup.module.scss

```diff
@@ -1,17 +1,42 @@
 .SidebarGroup {
-  position: relative;
-  border-bottom: 1px solid #5c5c5c;
-  padding: 6px 0;
-  &:last-of-type {
-    border-bottom: none;
-  }
+	position: relative;
+	padding: 6px 10px;
+	border-bottom: solid 1px #5c5c5c;
+	&:last-of-type {
+		border-bottom: none;
+	}
+
+	.title {
+		position: relative;
+		font-family: var(--font-primary);
+		font-size: 15px;
+		font-style: normal;
+		font-weight: 600;
+		color: #fff;
+		padding: 6px 0;
+	}
 }
-.title {
-  position: relative;
-  font-family: var(--font-primary);
-  font-size: 15px;
-  font-style: normal;
-  font-weight: 600;
-  color: #fff;
-  padding: 6px 0;
+.SidebarGroup.withDot {
+	--left-border-dot-top: 22px;
+	border-bottom: none;
+
+	&::after {
+		content: '';
+		position: absolute;
+		top: calc(var(--left-border-dot-top) - 5px);
+		left: -4px;
+		width: 9px;
+		height: 9px;
+		background-color: grey;
```

Comment:
@elusiveuphoria we should always be using our css vars for colors, I am sure you can find these colors in our global theme file

Reply:
I'm behind on the replies... resolved 1b5df3e9716d664d8d2cba3b3040e99673595322 I think...